/scratch365/mcarlozo/Toy_Problem/GP_Validation.py:407: RuntimeWarning: invalid value encountered in log
  plt.scatter(iter_space,np.log(GP_mean), label = r'$\mathbf{log(e(\theta))_{model}}$', s=100 )
/scratch365/mcarlozo/Toy_Problem/GP_Validation.py:438: RuntimeWarning: invalid value encountered in log
  csv_item_list = [iter_space, np.log(GP_mean), np.log(sse_sim)]
Date and Time:  29-Nov-2022 (11:14:17)
Emulator = True
Objective Function = obj
SSE Total =  1.2275e+02
Emulator = False
Objective Function = obj
Objective Function = LN_obj
